src/code/playback.c

```diff
@@ -0,0 +1,786 @@
+#include <ultra64.h>
+#include <global.h>
+
+#ifdef NON_MATCHING
+// 630 score, primarily regalloc left. Issues seem based around the usage of pan.
+// Currently the array of pan and reverb solves a lot, but doesn't make a ton of logical sense.
+void note_set_vel_pan_reverb(Note* note, NoteSubEu* sub, Reverb* reverb) {
```

Comment:
This does not fit our naming scheme. 
The suggestion would be something that fits our naming scheme, 
```suggestion
void Playback_NoteVelPanReverbSet(Note* note, NoteSubEu* sub, Reverb* reverb) {
```